nuxt.config.js

```diff
@@ -1,61 +1,104 @@
-        integrity: 'sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg',
-        crossorigin: 'anonymous'
+      {
+        rel: "apple-touch-icon",
+        sizes: "180x180",
+        href: "/icons/apple-touch-icon.png"
+      },
+      {
+        rel: "icon",
+        type: "image/png",
+        sizes: "32x32",
+        href: "/icons/favicon-32x32.png"
+      },
+      {
+        rel: "icon",
+        type: "image/png",
+        sizes: "16x16",
+        href: "/icons/favicon-16x16.png"
+      },
+      { rel: "icon", type: "manifest", href: "/icons/site.webmanifest" },
+      {
+        rel: "mask-icon",
+        href: "/icons/safari-pinned-tab.svg",
+        color: "#5bbad5"
+      },
+      { rel: "shortcut icon", href: "/icons/favicon.ico" },
+      { rel: "icon", type: "image/x-icon", href: "/icons/favicon.ico" },
+      {
+        rel: "stylesheet",
+        href:
+          "https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
+      },
+      {
+        rel: "stylesheet",
+        href: "https://fonts.googleapis.com/css?family=Lato"
+      },
+      {
+        rel: "stylesheet",
+        href: "https://use.fontawesome.com/releases/v5.0.10/css/all.css",
+        integrity:
+          "sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg",
+        crossorigin: "anonymous"
       }
     ]
   },
   plugins: [
-    '~/plugins/vuetify',
-    '~/plugins/eventBus.js',
-    '~/plugins/i18n.js',
-    '~/plugins/vue-moment.js'
+    "~/plugins/vuetify",
+    "~/plugins/eventBus.js",
+    "~/plugins/i18n.js",
+    "~/plugins/vue-moment.js"
   ],
   css: [
-    '~/assets/css/app.styl',
-    '~/assets/css/main.scss',
-    'flag-icon-css/css/flag-icon.min.css',
+    "~/assets/css/app.styl",
+    "~/assets/css/main.scss",
+    "flag-icon-css/css/flag-icon.min.css"
   ],
   modules: [
-    ['@nuxtjs/google-analytics', {
-      id: 'UA-65309624-3'
-    }],
-    ['storyblok-nuxt', {accessToken: 'plc0fBh2no8owEeTALSN2wtt', cacheProvider: 'memory'}],
+    [
+      "@nuxtjs/google-analytics",
+      {
+        id: "UA-65309624-3"
+      }
+    ],
+    [
+      "storyblok-nuxt",
+      { accessToken: "plc0fBh2no8owEeTALSN2wtt", cacheProvider: "memory" }
+    ],
+    "@nuxtjs/axios",
+    "@nuxtjs/proxy"
   ],
-  loading: {color: '#3B8070'},
+  loading: { color: "#3B8070" },
   build: {
-    vendor: ['vue-i18n'],
-    extend(config, {isDev, isClient}) {
+    vendor: ["vue-i18n"],
+    extend(config, { isDev, isClient }) {
       if (isDev && isClient) {
         config.module.rules.push({
-          enforce: 'pre',
+          enforce: "pre",
           test: /\.(js|vue)$/,
-          loader: 'eslint-loader',
+          loader: "eslint-loader",
           exclude: /(node_modules)/
-        })
+        });
       }
     }
+  },
+  proxy: {
+    "/meetups": {
+      target: "https://api.meetup.com/VueVixens/events/?&sign=true"
```

Comment:
**comment**: I tried to pass HTTP params here as well to get `event_hosts` shown in the response, no success 😓

Reply:
yeah the URI to get the avatar would be https://api.meetup.com/VueVixens/events/${EVENT_ID}/hosts?&sign=true&photo-host=public but I have no idea how to pass ${EVENT_ID} to the proxy config dynamically. Another approach could be to move the api lookup & processing to the CMS and have it passed in to asyncdata as as part of the page data but I don’t know if that’s possible with Storyblok.

Reply:
Nope, unfortunately Storyblok is not working as web proxy :(